cartographer/mapping/internal/2d/pose_graph_2d.cc

```diff
@@ -640,6 +702,36 @@ void PoseGraph2D::RunOptimization() {
   data_.global_submap_poses_2d = submap_data;
 }
 
+bool PoseGraph2D::ShouldSkipTrajectoryWorkItem(int trajectory_id) {
```

Comment:
Renaming to `bool PoseGraph2D::CanModify(int trajectory_id)` and negating the logic would be more readable in my opinion.

Reply:
Thanks! I named it CanAddWorkItemModifying, hope this is also fine.

Reply:
Looks better. Don't know if this method could be useful for anything else than adding work items though.